cuda/common/include/pcl/cuda/cutil_inline_runtime.h

```diff
@@ -177,6 +182,7 @@ inline int cutGetMaxGflopsGraphicsDeviceId()
 	current_device = 0;
 	while( current_device < device_count ) {
 		cudaGetDeviceProperties( &deviceProp, current_device );
+        int sm_per_multiproc = 0;
```

Comment:
In this case you still have an unused value warning, as `0` will never been used. So change this to:
```
		int sm_per_multiproc = 1;
		if (deviceProp.major != 9999 || deviceProp.minor != 9999) {
			sm_per_multiproc = _ConvertSMVer2Cores(deviceProp.major, deviceProp.minor);
		}
```

Reply:
I missed that. A much better solution.